scopes/harmony/aspect-loader/aspect-loader.main.runtime.ts

```diff
@@ -481,7 +483,15 @@ export class AspectLoaderMain {
     return updatedManifest;
   }
 
+  getPluginDefsPatterns() {
+    const patternsSet = new Set();
+    this.pluginSlot.values().flatMap((val) => val.map((def) => patternsSet.add(def.pattern)));
+    const uniquePatterns = [...patternsSet];
+    return uniquePatterns;
+  }
+
   getPluginDefs() {
+    this.getPluginDefsPatterns();
```

Comment:
Why do you need that line here? You are not using the result

Reply:
sorry leftovers from some testing I was doing